tools/ci_build/build.py

```diff
@@ -35,6 +35,12 @@ class UsageError(BaseError):
     def __init__(self, message):
         super().__init__(message)
 
+# According to the BUILD.md, python 3.5+ is required:
+if sys.version_info[0] != 3 :
+	raise BuildError("Bad python major version: expecting python 3, found version '{}'".format(sys.version))
+if sys.version_info[1] < 5 :
+	raise BuildError("Bad python minor version: expecting python 3.5+, found verison '{}'".format(sys.version))
```

Comment:
>verison [](start = 74, length = 7)

s/verison/version/

Could be version_info[0] < 3.

Spaces not tabs please.

Instead of inline statements can this check be in a function that's called from main() so it's more consistent with the existing code?